server/handlers/getters.go

```diff
@@ -104,25 +104,21 @@ func (ah *ActivityHandler) getMessageAndChatFromActivityIds(activityIds msteams.
 	return msg, nil, nil
 }
 
-func (ah *ActivityHandler) getOrCreateSyntheticUser(userID, displayName string) (string, error) {
-	mmUserID, err := ah.plugin.GetStore().TeamsToMattermostUserID(userID)
+func (ah *ActivityHandler) getOrCreateSyntheticUser(user *msteams.User, tryCreate bool) (string, error) {
```

Comment:
tryCreate can we change the name for this variable to make this more descriptive ?